src/server/index.ts

```diff
@@ -0,0 +1,25 @@
+import Fastify from 'fastify';
+import { fastifyAutoload } from '@fastify/autoload';
+import { join } from 'node:path';
+import { config } from '@/config/env';
+import { prettyLog } from '@/utils/prettyLog';
+
+export async function buildServer() {
+  const server = Fastify({
+    logger: {
+      transport: prettyLog,
```

Comment:
One doubt... This piece is creating a log struct with a configuration that prettify logs?

Reply:
yep, if you see the `prettyLog` import it is the instance of a logger prettify config